### DISCUSSION TITLE
Stack Trace - Red Error - let me CTRL+c copy this

### DISCUSSION THREAD

**Discussion by shak2** [godotengine/godot-proposals#9874] [Editor]
When a game crashes out, the stack trace has some red text, let me highlight, right click, copy that. or highlight CTRL+c that.

Currently, typing it in manually into google every time.

**Comment by Calinou** [comment]
Can you take a screenshot of which red message you're referring to?

**Reply by shak2** [reply]
I am not allowed to screenshot, but to replicate you can add an extra letter to a variable then when the game crashes it will have a "Stack Trace" tab on the output bottom window saying there is a "Parse Error: Identifier not declared in the current scope".

The red text i'd like to be able to copy is that parse error line.

**Reply by AThousandShips** [reply]
Not allowed to screenshot? You can use an empty project if the project you're working on is secret.

But if it's the stack trace you're talking about that might already be a proposal for that but can't find it at the moment

**Reply by Zireael07** [reply]
Yeah I'm like 90% certain there's already a proposal for that

**Reply by AThousandShips** [reply]
Also there's a little copy button next to the breakpoint button (red circle) that copies an error message

**Reply by dalexeev** [reply]
Related:

* #6998
* godotengine/godot#77776

Currently, you can expand the error/warning panel and copy the message from there:

![](https://github.com/godotengine/godot-proposals/assets/47700418/b1174200-da63-4d8a-8f5d-39986182295c)
